app/updates/KinesisEventSender.scala

```diff
@@ -32,7 +32,7 @@ object KinesisEventSender extends ThriftSerializer {
     kinesisClient
   }
 
-  def putCapiUpdate(collectionId: String, collectionJson: CollectionJson): Unit = {
+  def putCapiUpdate(collectionId: String, collectionJson: CollectionJson, isDelete: Boolean): Unit = {
```

Comment:
I'm not sure I like `isDelete`, it doesn't work nicely when we want to send other actions. Why not creating a new method? `putCapiDelete` or something like that. This new method maybe doesn't require an empty `CollectionJson`